src/app/item-page/simple/field-components/file-section/total-downloads.component.html

```diff
@@ -0,0 +1,7 @@
+<ds-metadata-field-wrapper [label]="label | translate">
+  <div *ngIf="totalDownloads !== null" class="total-downloads-section">
```

Comment:
`null` is only if an error?
When there are no downloads is this displayed with download = 0?